### PR TITLE
Add license information

### PR DESCRIPTION
**Pull request by dawehner**
Solves https://github.com/jsdrupal/drupal-admin-ui/issues/233 hopefully

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Most of this repository is licensed under [MIT](https://opensource.org/licenses/MIT),
+beside the admin_ui_support php module, which is licensed under [GPLv2+](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html), see the LICENSE.txt inside its folder.
```
Comment (justafish) [inline LICENSE.md]: s/beside the/except for
s/php/Drupal

Comment (dawehner) [reply LICENSE.md]: 👍 Thank you

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Most of this repository is licensed under [MIT](https://opensource.org/licenses/MIT),
```
Comment (mattgrill) [inline LICENSE.md]: Should we actually include the text of MIT license?

Comment (dawehner) [reply LICENSE.md]: That's a good question

File: package.json [hunk 3]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "vfancy",
   "version": "0.1.0",
+  {
```
Comment (fafnirical) [inline package.json]: This looks invalid. Did you mean `"license": "MIT"` (without the curly brackets)?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "vfancy",
   "version": "0.1.0",
+  "license" : "MIT"
```
Comment (justafish) [inline package.json]: :eyes: 

```,```

**Pull request merged**